app/views/home/index.html.slim

```diff
@@ -47,5 +47,8 @@
     - else
       .row
         .col-md-offset-4.col-md-4.col-sm-offset-3.col-sm-6.col-xs-12
-          = link_to "Subscribe to our newsletter", "#", class: "btn btn-pink btn-block btn-lg"
+          = form_tag('/subscription', method: "post", id: "subscribe") do 
```

Comment:
last thing - sorry but I haven't noticed it earlier.  Maybe we should submit this form remotely? so the page won't be reloaded on submit?